src/Mod/Fem/femmesh/gmshtools.py

```diff
@@ -644,6 +645,8 @@ def get_boundary_layer_data(self):
             Console.PrintMessage(f"  {self.bl_setting_list}\n")
 
     def write_groups(self, geo):
+        reg_exp = re.compile(r"(?:.*\.)?(?P<shape>Solid|Face|Edge|Vertex)(?P<index>\d+)$")
```

Comment:
A short explanation (or a couple of examples) would help later readers understand this regex more quickly.

Reply:
Done

Reply:
posible ==> possible